vembrane/modules/annotate.py

```diff
@@ -0,0 +1,129 @@
+import contextlib
+import csv
+import sys
+import yaml
+import pandas as pd
+
+from sys import stderr
+from typing import Iterator, List, Optional
+
+import asttokens
+from pysam.libcbcf import VariantFile, VariantRecord
+
+from ..common import check_expression
+from ..errors import VembraneError, HeaderWrongColumnNumber
+from ..representations import Environment
+
+from itertools import chain
+import numpy as np
+
+
+def add_subcommmand(subparsers):
+    parser = subparsers.add_parser("annotate")
+    parser.add_argument(
+        "config",
+        type=check_expression,
+        help="The configuration file.",
+    )
+    parser.add_argument(
+        "vcf", help="The file containing the variants.", nargs="?", default="-"
+    )
+
+
+def annotate_vcf(
+    vcf: VariantFile,
+    expression: str,
+    ann_key: str,
+    ann_data: dict,
+    config: dict,
+) -> Iterator[tuple]:
+
+    env = Environment(expression, ann_key, vcf.header)
+
+    current_chrom = None
+    current_index = None
+    current_data = None
+    indices = []
+
+    record: VariantRecord
+    for idx, record in enumerate(vcf):
+        if not current_chrom == record.chrom:
+            current_chrom = record.chrom
+            chrom = "chr" + record.chrom
```

Comment:
CHROM may or may not start with "chr" depending on the reference used / contigs defined etc. I vaguely remember reading something about "alternative names" for contig in either the SAM or the VCF specification, we should probably leverage those.

Reply:
there should be a parameter "add_chrom_to_bed" and "remove_chrom_from_bed", or better detect it automatically

Reply:
i implemented some kind of automatic detection